Source/WebCore/loader/ResourceMonitor.cpp

```diff
@@ -41,7 +41,8 @@ namespace WebCore {
 
 #if ENABLE(CONTENT_EXTENSIONS)
 
-#define RESOURCEMONITOR_RELEASE_LOG(fmt, ...) RELEASE_LOG(ResourceLoading, "%p - ResourceMonitor(frame %p)::" fmt, this, m_frame.get(), ##__VA_ARGS__)
+#define RESOURCEMONITOR_RELEASE_LOG(fmt, ...) RELEASE_LOG(ResourceMonitoring, "%p - ResourceMonitor(frame %p)::" fmt, this, m_frame.get(), ##__VA_ARGS__)
```

Comment:
Would it be OK to log the frame ID instead of the frame pointer? Also, is it required to log the pointer of the ResourceMonitor? This will increase the log volume, and could be good to avoid, if possible.

Reply:
Thanks! I'll fix that.